libs/tree-sitter-wing/grammar.js

```diff
@@ -210,6 +210,7 @@ module.exports = grammar({
         "if",
         field("condition", $.expression),
         field("block", $.block),
+        optional(seq("elif", field("elif_condition", $.expression), field("elif_block", $.block))),
```

Comment:
Missing a `repeat` here for multiple `elifs`, something like:
```suggestion
        repeat(seq("elif", field("elif_condition", $.expression), field("elif_block", $.block))),
```

Reply:
You're right, I already saw that, but I think I have to create an `elif_statement` to iterate, something like that

```js
    if_statement: ($) =>
      seq(
        "if",
        field("condition", $.expression),
        field("block", $.block),
        repeat($.elif_statement),
        optional(seq("else", field("else_block", $.block)))
      ),

    elif_statement: ($) =>
      seq(
        "elif",
        field("condition", $.expression),
        field("block", $.block),
      ),

```